src/common/plugin.ts

```diff
@@ -45,12 +47,25 @@ export abstract class Plugin implements IPlugin {
     const channelName = channel.name;
     const categoryName = channel.parent?.name.toLowerCase();
 
-    if (this.pluginChannelName && this.pluginChannelName !== channelName) {
-      const id = this.container.guildService.getChannel(this.pluginChannelName).id;
-      return `Please use this command in the <#${id}> channel.`;
+    const allowedChannels = [...(this.pluginChannelNames ?? [])];
+    if (this.pluginChannelName) {
+      allowedChannels.push(this.pluginChannelName);
     }
 
-    if (this.pluginCategoryName && this.pluginCategoryName.toLowerCase() !== categoryName) {
+    if (allowedChannels.length !== 0) {
```

Comment:
So where are you actually checking that the channelName is in the allowedChannels?

Reply:
oh LOL that line escaped somehow, added back